### PR TITLE
DOCS-2281: Install a Debian VM on Scale

### PR DESCRIPTION
**Pull request by BonnieF**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/Virtualization/CreatingManagingVMsSCALE.md [hunk 1]
```diff
@@ -82,3 +82,143 @@ If the VM you created has no Guest OS installed, The VM *State* toggle and <i cl
...
+
+**Confirm Options:**
+* Verify the information is correct and then click the Submit button.
+
+{{< /expand >}}
+
+
+* Once the VM has been created, expand the VM by clicking on the down pointing arrow to the right of the new VM. Click the Start button.
+
+* Click the Display button to step through the Debian installation.
```
Comment (Mrt134) [inline content/SCALE/Virtualization/CreatingManagingVMsSCALE.md:145]:
```suggestion
* Click *Display* to open a virtual monitor to the VM and see the Debian Graphical Installation screens.
```

File: content/SCALE/Virtualization/CreatingManagingVMsSCALE.md [hunk 2]
```diff
@@ -82,3 +82,143 @@ If the VM you created has no Guest OS installed, The VM *State* toggle and <i cl
...
+* Click the Next button.
+
+
+**Confirm Options:**
+* Verify the information is correct and then click the Submit button.
+
+{{< /expand >}}
+
+
+* Once the VM has been created, expand the VM by clicking on the down pointing arrow to the right of the new VM. Click the Start button.
```
Comment (Mrt134) [inline content/SCALE/Virtualization/CreatingManagingVMsSCALE.md:143]:
```suggestion
* Once the VM has been created, start it by expanding the VM entry (down pointing arrow to the right of the VM name) and click *Start*.
```

File: content/SCALE/Virtualization/CreatingManagingVMsSCALE.md [hunk 3]
```diff
@@ -82,3 +82,143 @@ If the VM you created has no Guest OS installed, The VM *State* toggle and <i cl
...
+
+* Click the Display button to step through the Debian installation.
+
+{{< expand "Debian Install Example." "v" >}}
+
+**Debian Graphical Install**
+* Press the Return key, on the keyboard, to start the Debian Graphical Install.
```
Comment (Mrt134) [inline content/SCALE/Virtualization/CreatingManagingVMsSCALE.md:150]:
```suggestion
* Press <kbd>Return</kbd> to start the Debian Graphical Install.
```

**Review by tonyriv3 — APPROVED** [review]
I changed some passive voice in the `.part` file to active voice. I also edited the *ADD* button in the main article to obey the style guide. The rest looks good to me.

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**